.github/workflows/ci_cross.yml

```diff
@@ -71,7 +71,7 @@ jobs:
       - name: v_win.c can be compiled and run with -os windows
         run: |
           ./v -os windows -o /tmp/v_win.c cmd/v
-          x86_64-w64-mingw32-gcc  /tmp/v_win.c -std=c99 -w -municode -o v_from_vc.exe
+          x86_64-w64-mingw32-gcc /I .\thirdpart\stdatomic\win /tmp/v_win.c -std=c99 -w -municode -o v_from_vc.exe
```

Comment:
Should that be `thirdparty`?  Why backslashes on the new option, forward slashes on the rest?  I think the mingw stuff is fine with forward slashes everywhere.